docs/references/uvh5_memo.tex

```diff
@@ -165,8 +165,10 @@ \subsection{Required Parameters}
 \item \textbf{Ntimes}: \textit{int} The number of time samples present in the
   data. (\textit{Ntimes})
 \item \textbf{uvw\_array}: \textit{float} An array of the uvw-coordinates
-  corresponding to each observation in the data. This is a two-dimensional array of
-  size (Nblts, 3). Units are in meters. (\textit{uvw\_array})
+  corresponding to each observation in the data. Baselines are specified as from
+  \verb+ant_1_array+ to \verb+ant_2_array+, inferring the position of antenna 2
```

Comment:
```suggestion
  \verb+ant_1_array+ to \verb+ant_2_array+, implying the position of antenna 2
```

Reply:
Happily... I was sure I'd gotten it right but in revision this feels like a bit of a grey area for nit-picking over it 🤷... I followed the following from an [IEEE guide (Sec. 3, Words Often Confused)](http://journals.ieeeauthorcenter.ieee.org/wp-content/uploads/sites/7/IEEE-Editorial-Style-Manual-for-Authors.pdf):
```
Imply: something suggested though not expressed.
Infer: something deduced from evidence.
```

Reply:
You're right, I'm definitely being pedantic, we could have left it the way it was. Thanks for making the change anyway.